2021-12-13 10:43:34,174:	ERROR:	Mqtt.py:connect():69	Exception in connect to mqtt: [Errno -3] Temporary failure in name resolution
2021-12-13 14:13:09,193:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:10,250:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:11,305:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:12,360:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:13,416:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:14,471:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:15,528:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:16,582:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:17,638:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:18,691:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:19,746:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:20,802:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:21,859:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:22,865:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:23,920:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:25,026:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:26,082:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:27,089:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:28,151:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:29,206:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:30,263:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:31,328:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:32,435:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:33,444:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:34,554:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:13:35,573:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:28,344:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:29,400:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:30,456:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:31,562:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:32,567:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:33,622:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:34,682:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:35,739:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:36,799:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:37,857:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:38,913:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:39,920:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:40,978:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:42,035:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:43,092:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:44,199:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:45,205:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:46,263:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:47,370:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:48,376:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:49,434:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:50,490:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:51,596:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:52,602:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:53,659:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:54,766:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:55,772:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:56,828:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:57,933:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:58,939:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:33:59,996:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:01,053:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:02,108:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:03,164:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:04,272:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:05,277:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:06,333:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:07,388:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:08,444:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:09,500:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:10,555:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:11,611:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:13,723:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:14,729:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:15,781:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:16,886:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:17,889:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:18,994:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:20,001:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:21,057:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:22,118:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:23,176:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:24,241:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 14:34:25,348:	ERROR:	MqttDataHandler.py:__handler_cloud_to_device_request_topic():76	mqtt data receiver in topic v1/devices/me/telemetry invalid
2021-12-13 15:38:19,254:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:42:19,256:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:42:23,126:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:43:48,612:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:44:31,451:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:46:10,845:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:46:31,075:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:47:47,961:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:50:41,984:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:51:00,934:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:51:27,456:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:53:26,224:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:53:44,208:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:56:42,520:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:58:29,513:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:58:37,303:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:58:48,251:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:59:46,972:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 15:59:51,760:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:00:45,725:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:00:48,992:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:06:23,425:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:11:59,056:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:12:06,835:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0]
2021-12-13 16:13:23,476:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:13:28,944:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0]
2021-12-13 16:14:32,809:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:14:40,939:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0]
2021-12-13 16:17:23,583:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:17:31,372:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0]
2021-12-13 16:21:41,359:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:21:51,448:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0]
2021-12-13 16:21:55,875:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:22:22,434:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0]
2021-12-13 16:27:28,447:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:27:39,339:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0]
2021-12-13 16:27:57,466:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1]
2021-12-13 16:36:57,586:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1, 13, 10]
2021-12-13 16:37:02,454:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0, 13, 10]
2021-12-13 16:37:04,913:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1, 13, 10]
2021-12-13 16:37:13,298:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0, 13, 10]
2021-12-13 16:37:19,531:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 1, 13, 10]
2021-12-13 16:37:27,364:	INFO:	UART.py:send():40	send uart: [0, 5, 2, 0, 0, 13, 10]
